tests/console/validate_raid_encrypt_home.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate that a RAID multi-disk is directly formatted with an
+# encrypted home (md is not partitioned).
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use Mojo::JSON qw(decode_json);
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+sub run {
+    select_console 'root-console';
+    my $md = get_test_suite_data()->{mds}[1];
+
+    my $lsblk_output = decode_json(script_output("lsblk -M -J /dev/$md->{name}"));
+    my $dev_name = $lsblk_output->{blockdevices}[0]{name};
+    my $dev_blockdevices = $lsblk_output->{blockdevices}[0];
```

Comment:
you don't need this intermediate steps, kind of abuse of assignations in general (as I mentioned previously).

Reply:
Updated.